prysm/assetlist.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://github.com/cosmos/chain-registry/blob/master/assetlist.schema.json",
+  "chain_name": "Prysm Network",
```

Comment:
the chain_name for prysm must be "prysm" (all lowercase). It is only used for indexing at the Chain Registry, and never actually seen by app users.